app/src/androidTest/java/ch/epfl/sdp/blindly/profile_setup/TestProfileOrientation.kt

```diff
@@ -90,10 +91,10 @@ class TestProfileOrientation {
         val buttonContinue = onView(withId(R.id.button_p5))
         buttonContinue.perform(click())
 
-        intended(hasComponent(ProfileShowMe::class.java.name))
-        Intents.release()
+        val TEST_SEXUAL_ORIENTATIONS = arrayListOf<String>("Straight", "Lesbian", "Gay")
```

Comment:
Shouldn't it be a const val outside of the class ? (Like in `TestProfileBirthday`)

Reply:
Yep took not of it, it's in the next branch fix